front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -242,7 +242,7 @@ export function AgentMessage({
   }, [
     agentMessageToRender.content,
     agentMessageToRender.status,
-    agentMessageToRender.action,
+    agentMessageToRender.actions,
```

Comment:
are we sure this is still fine ? Any insight on why this thing is a dependency even though not used above ? Could This be replaced by actions.length or do we mutate the action's content ?